sdk/src/settings.rs

```diff
@@ -218,40 +233,327 @@ impl Default for Verify {
 
 impl SettingsValidate for Verify {}
 
+// TODO: thumbnails/previews for audio?
```

Comment:
Seems like this belongs with thumbnail code and not the settings.

Reply:
Agreed.